app/assets/javascripts/controllers/container_dashboard/util/charts-mixin.js

```diff
@@ -1,6 +1,16 @@
 angular.module('miq.util').factory('chartsMixin', function() {
   'use strict';
 
+  var hourlyTimeTooltip = function(d) {
+    var theMoment = moment(d[0].x);
+    return _.template('<table class="c3-tooltip">' +
```

Comment:
I'm ..not that fond of using tables for layout, is there any reason for that?

But, LGTM, :+1: for `<%-` :)